static/src/stylesheets/module/facia-garnett/item-layouts/_fc-item--full-media-100.scss

```diff
@@ -43,7 +38,7 @@ x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
     &:not(.fc-item--has-cutout) {
         .fc-item__image-container {
             display: block;
-            padding-bottom: 40%;
+            padding-bottom: 60%;
```

Comment:
We're removing the thinning of images in fullMedia100. This makes the image much taller but isn't cropping them strangely.